account/src/main/java/com/whz/account/resource/StockAcquiredSubscriber.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2020, Fachgruppe Informatik WHZ <lationts@gmail.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.whz.account.resource;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.DeliverCallback;
+import io.vlingo.actors.Logger;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeoutException;
+
+public enum StockAcquiredSubscriber {
+  INSTANCE;
+
+  private final Logger logger = Logger.basicLogger();
+
+  private static final String EXCHANGE_NAME = "logs"; // ???
```

Comment:
What does `???` mean?

Reply:
Die Klasse ist noch in Arbeit. Flo hat 2x denselben exchange_name für 2 verschiedene exchanges genommen. Ich dachte mir, dass das vermutlich falsch ist, weil ich für jeden pub-sub einen eigenen exchange aufgemacht hätte. Kann aber sein, dass es auch sinnvoll ist, das man alle pubs-subs über nur ein und denselben exchange macht, solange der exchange-type derselbe ist.